app/org/sagebionetworks/bridge/models/schedules/ABTestScheduleStrategy.java

```diff
@@ -103,13 +103,13 @@ public void validate(Errors errors) {
             percentage += group.getPercentage();
         }
         if (percentage != 100) {
-            errors.reject(String.format("groups in AB test plan add up to %s\u0025 and not 100\u0025 (give 20\u0025 as 20, for example)", percentage));
+        	errors.rejectValue("scheduleGroups", "groups must add up to 100\u0025 (not "+percentage+"\u0025; give 20\u0025 as 20, for example)");
```

Comment:
This is actually super-subtle, but unicode escapes (\uXXXX) in Java is actually a compiler escape, not a string escape. (See http://javajee.com/unicode-escapes-in-java). The original bug was because the attempt to use \u0025 to bypass '%' in String.format() didn't actually work (they should have used "%%", so we were getting garbage string formats. Now that we're not using String.format(), \u0025 isn't necessary (not that it ever worked), and actually makes the string harder to read. Recommend changing them to just %.